charts/csi-powermax/values.yaml

```diff
@@ -323,6 +338,10 @@ csireverseproxy:
   # Default value: None
   # Examples: "1111", "8080"
   port: 2222
+  # Name of the reverseproxy secret as a Kubernetes resource.
+  # Default value: "powermax-reverseproxy-secret"
+  # Example: "powermax-reverseproxy-secret"
+  secretName: powermax-reverseproxy-secret
```

Comment:
This value can be empty by default to support backwards compatibility with the ConfigMap. If a customer upgrades without creating a Secret, this will attempt to mount the Secret.